setup.py

```diff
@@ -48,15 +51,16 @@ def run(self):
     url="https://github.com/allegro/bigflow",
     packages=setuptools.find_packages(exclude=('test', 'e2e')),
     data_files=[
-        ('requirements', ['requirements/base.txt', 'requirements/stackdriver_extras.txt']),
+        ('requirements', ['requirements/base.txt', 'requirements/stackdriver_extras.txt', 'requirements/bigquery_extras.txt']),
```

Comment:
Lets change "stackdriver" to more generic "monitoring" (in the whole project).

Reply:
Done